Automattic/wp-calypso
# Domains: Fix price rule for domain mapping step

#### Changes proposed in this Pull Request

This PR fixes an issue with the domain mapping step, where we show the wrong price for the mapping, since it will be bundled with a plan in the cart.

#### Testing instructions

- For a site on free plan, go to `/domains/add/mapping/FREE_SITE_SLUG?initialQuery=example.com`
- Before applying the fix, you should see a price above the Add button
- After applying the fix, you should see Included in paid plans, as shown on this screenshot:

<img width="243" alt="Screenshot 2020-05-13 at 4 05 19 PM" src="https://user-images.githubusercontent.com/13062352/81822909-f7b61f00-9533-11ea-8638-76aece489117.png">
